plugin/src/main/kotlin/spp/jetbrains/sourcemarker/view/window/LiveViewChartWindow.kt

```diff
@@ -99,19 +99,25 @@ class LiveViewChartWindow(
         val stop = Instant.now()
         val start = stop.truncatedTo(ChronoUnit.SECONDS)
             .minusSeconds(60 + (getHistoricalMinutes() * 60).toLong())
-        viewService.getHistoricalMetrics(
-            liveView.entityIds.toList(),
-            liveView.viewConfig.viewMetrics,
-            step, start, stop, labels
-        ).onSuccess {
-            for (i in 0 until it.data.size()) {
-                val stepBucket = step.bucketFormatter.format(start.plusSeconds((step.seconds * i).toLong())).toLong()
-                val metricData = it.data.getJsonObject(i)
-                addMetric(metricData.put("timeBucket", stepBucket))
+        viewService.clearLiveViews()
```

Comment:
This is a fairly destructive act. Why can't you remove the single live view causing issues?

Reply:
How is this resolved? You're still using `clearLiveViews()`.

Reply:
Sorry I thought I have already taken it out.